doc/DESCRIPTION.md

```diff
@@ -0,0 +1,88 @@
+DESCRIPTION
+
```

Comment:
I might add some schema here:
scenario 1: VM to VM in the same VNI on the same host
```
          /----[Firewall ?]--[Antispoof]--[ sniffer ?]--[vhost, tap or bench brick]
[switch]<
          \----[Firewall ?]--[Antispoof]--[ sniffer ?]--[vhost, tap or bench brick]
```

Reply:
I'll do other schema latter

Reply:
scenario 2: VM to VM in the same VNI on diferent host
```
           /- [nic] -- [sniffer ?] -- [vtep] -- [firewall?] -- [antispoof] -- [sniffer ?] -- [vhost, tap or bench brick]
{The WEB} <
           \- [nic] -- [sniffer ?] -- [vtep] -- [firewall?] -- [antispoof] -- [sniffer ?] -- [vhost, tap or bench brick]
```

Reply:
scenario 3: VM to VM in the same VNI on diferent host with more than 1 VN of the same VNI on 1rst host
```
                                                          /- [firewall?] -- [antispoof] -- [sniffer ?] -- [vhost, tap or bench brick]
           /- [nic] -- [sniffer ?] -- [vtep] -- [switch] {- [firewall?] -- [antispoof] -- [sniffer ?] -- [vhost, tap or bench brick]
{The WEB} <
           \- [nic] -- [sniffer ?] -- [vtep] -- [firewall?] -- [antispoof] -- [sniffer ?] -- [vhost, tap or bench brick]
```